web/package.json

```diff
@@ -78,7 +79,8 @@
   "jest": {
     "transformIgnorePatterns": [
       "/node_modules/(?!antd|@ant-design|rc-.+?|@babel/runtime).+(js|jsx)$"
-    ]
+    ],
+    "testMatch": [ "<rootDir>/src/hooks/**/*.test.tsx" ]
```

Comment:
Shouldn't this match everything not only hooks?

Reply:
my bad